extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -83,8 +93,12 @@ export class BindExpression {
     /** @const {string} */
     this.expressionString = expressionString;
 
-    /** @const {!./bind-expr-defines.AstNode} */
+    /** @const @private {!./bind-expr-defines.AstNode} */
     this.ast_ = parser.parse(this.expressionString);
+
+    /** @const @private {!Object} */
+    this.builtInScope_ = Object.create(null);
+    this.builtInScope_['Math'] = Math;
```

Comment:
That is already something you can do in JS and it might be counter-intuitive to have to prefix a call to Math.random in bind when you don't have to in JS. 

I imagine we'll have the complete list of all these built-in scopes published somewhere. If users end up overwriting one then it's likely they're not going to use ours anyway e.g. would someone naming their state "Math" reasonably expect that certain uses of Math will refer to the custom state, and others to our built-in?